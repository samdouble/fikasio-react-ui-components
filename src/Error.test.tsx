import React from 'react';
import renderer from 'react-test-renderer';
import { Error } from '.';

describe('Error', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(
      <Error>
        This is an error
      </Error>
    ).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
