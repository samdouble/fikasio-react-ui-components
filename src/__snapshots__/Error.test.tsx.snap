// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Renders correctly 1`] = `
<div
  className="fikasio-error fikasio-theme-dark"
  style={Object {}}
>
  <div
    className="fade alert alert-danger show"
    role="alert"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-triangle-exclamation fa-lg "
      data-icon="triangle-exclamation"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={
        Object {
          "fontSize": 16,
          "marginRight": 10,
        }
      }
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M256 32c14.2 0 27.3 7.5 34.5 19.8l216 368c7.3 12.4 7.3 27.7 .2 40.1S486.3 480 472 480L40 480c-14.3 0-27.6-7.7-34.7-20.1s-7-27.8 .2-40.1l216-368C228.7 39.5 241.8 32 256 32zm0 128c-13.3 0-24 10.7-24 24l0 112c0 13.3 10.7 24 24 24s24-10.7 24-24l0-112c0-13.3-10.7-24-24-24zm32 224a32 32 0 1 0 -64 0 32 32 0 1 0 64 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
